table/tables/tables.go

```
@@ -584,12 +575,14 @@ func (t *Table) removeRowIndices(ctx context.Context, h int64, rec []types.Datum
 		vals, err := v.FetchValues(rec)
```

The error is ignored